app/assets/stylesheets/patternfly_and_overrides.scss

```diff
@@ -315,3 +315,7 @@ a.btn-info span{
   max-height: 80vh;
   overflow-y: auto;
 }
+// Color origin: pf-orange-400
+.pficon-ok.warn:before {
```

Comment:
my main concern this effects all pficon-ok.warn across the app, not sure if its intended or not.